charts/spire/charts/spire-server/values.yaml

```diff
@@ -266,8 +266,8 @@ nodeAttestor:
 
 # tornjak - Tornjak specific configuration
 tornjak:
-  # -- Deploys Tornjak API (backend)
-  enabled: false
+  # -- Deploys Tornjak API (backend) (Not for production)
+  enabled: false # (Not for production)
```

Comment:
I'd be ok with just the comment at 269 and not on the enabled line if it makes the linter more happy.